mkosi/__init__.py

```diff
@@ -2660,3 +2685,9 @@ def run_verb(args: MkosiArgs, images: Sequence[MkosiConfig]) -> None:
 
             if args.verb == Verb.serve:
                 run_serve(last)
+
+            if args.verb == Verb.journalctl:
+                return run_journalctl(args, last)
```

Comment:
Let's drop the return here and below since we don't do it for run_serve() either